packages/scandipwa/src/util/Product/Product.js

```diff
@@ -308,3 +308,18 @@ export const getExtensionAttributes = (product) => {
 
     return {};
 };
+
+/** @namespace Util/Product/sortBySortOrder */
+export const sortBySortOrder = (options, sortKey = 'sort_order') => options.sort(
+    // eslint-disable-next-line no-nested-ternary
+    (a, b) => (a[sortKey] < b[sortKey] ? -1 : (a[sortKey] > b[sortKey]) ? 1 : 0)
+);
+
+/** @namespace Util/Product/sortAlphabetically */
+export const sortAlphabetically = (options, key, caseSensitive = false) => options
+    .sort((a, b) => {
+        const textA = caseSensitive ? a[key] : a[key].toUpperCase();
+        const textB = caseSensitive ? b[key] : b[key].toUpperCase();
+        // eslint-disable-next-line no-nested-ternary
```

Comment:
Please don't disable eslint for no reason.
This can be implemented without the nested ternary operator.